### PR TITLE
Docs folder standardization and other fixes

### PR DESCRIPTION
**Pull request by luizcmarin**

### CODE REVIEW

File: docs/internals.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Internals
+
+## Unit testing
```
Comment (samdark) [inline docs/internals.md:3]: Testing section should be merged here.

**Review by samdark — CHANGES_REQUESTED** [review]

File: docs/guide/pt-BR/usage-standalone.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    ```shell
+    cp ./vendor/yiisoft/db-migration/bin/yii-db-migration.php ./yii-db-migration.php
+    ```
+
+2. Define DB connection in configuration file (see
+   [Yii DB documentation](https://github.com/yiisoft/db/blob/master/docs/en/README.md#create-connection)).
+   For example, MySQL connection:
+
+    ```php
```
Comment (s1lver) [inline docs/guide/pt-BR/usage-standalone.md:15]: Extra padding

File: docs/internals.md [hunk 3]
```diff
@@ -28,14 +28,10 @@ in the root directory of each package.
 
 For running the Docker containers, you can use the following command:
 
-```shell
+```dockerfile
```
Comment (s1lver) [inline docs/internals.md:31]: Why **dockerfile**?

File: docs/guide/pt-BR/usage-standalone.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+    'db' => new \Yiisoft\Db\Mysql\Connection(
+        new \Yiisoft\Db\Mysql\Driver('mysql:host=mysql;dbname=mydb', 'user', 'q1w2e3r4'),
+        new \Yiisoft\Db\Cache\SchemaCache(new \Yiisoft\Cache\ArrayCache()),
+    ),
+    ```
+
+3. Optionally, modify other options in the configuration file. Each option has a comment with description.
+4. Run the console command without arguments to see the list of available migration commands:
+
+    ```shell
```
[the remark this answers is not in the document]
Comment (luizcmarin) [reply docs/guide/pt-BR/usage-standalone.md:25]: just for the links to exist and function for now. When I start translating this package I will make a new copy of the English original to ensure I have the latest updates. For now, ignore the 'pt-BR' folder, please.

File: docs/guide/pt-BR/usage-standalone.md (hunk 2 above)
Comment (luizcmarin) [reply docs/guide/pt-BR/usage-standalone.md:15]: just for the links to exist and function for now. When I start translating this package I will make a new copy of the English original to ensure I have the latest updates. For now, ignore the 'pt-BR' folder, please.

File: docs/guide/pt-BR/usage-standalone.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
+# Standalone usage
+
+## With configuration file
+
+1. Copy configuration file `./vendor/yiisoft/db-migration/bin/yii-db-migration.php` to root folder of your project:
+
+    ```shell
```
[the remark this answers is not in the document]
Comment (luizcmarin) [reply docs/guide/pt-BR/usage-standalone.md:7]: just for the links to exist and function for now. When I start translating this package I will make a new copy of the English original to ensure I have the latest updates. For now, ignore the 'pt-BR' folder, please.

**Review by samdark — APPROVED** [review]

**Pull request merged**